vspace/src/test/java/edu/asu/diging/vspace/core/services/impl/SlideManagerTest.java

```diff
@@ -0,0 +1,147 @@
+import org.mockito.Mock;
+import org.mockito.Mockito;
+import org.mockito.MockitoAnnotations;
+
+import edu.asu.diging.vspace.core.data.SequenceRepository;
+import edu.asu.diging.vspace.core.data.SlideRepository;
+import edu.asu.diging.vspace.core.exception.SlideDoesNotExistException;
+import edu.asu.diging.vspace.core.model.ISlide;
+import edu.asu.diging.vspace.core.model.impl.Sequence;
+import edu.asu.diging.vspace.core.model.impl.Slide;
+
+public class SlideManagerTest {
+
+    @Mock
+    private SequenceRepository sequenceRepo;
+
+    @Mock
+    private SlideRepository slideRepo;
+
+    @InjectMocks
+    private SlideManager slideManagerToTest = new SlideManager();
+
+    // setting common used variables and Objects
+    private String slideId, moduleId, sequenceId, slideIdNotInSequence, sequenceIdOther;
+    private List<ISlide> slidesList = Arrays.asList(new Slide());
+    private Sequence sequenceObj;
+    private Sequence sequenceObjOther;
+
+    @Before
+    public void init() {
+        MockitoAnnotations.initMocks(this);
+
+        sequenceObj = new Sequence();
+        sequenceObjOther = new Sequence();
+
+        slideId = "SLI000000002";
+        slideIdNotInSequence = "SLI000000219";
+        moduleId = "MOD000000002";
+        sequenceId = "SEQ000000004";
+        sequenceIdOther = "SEQ000000005";
+        slidesList.get(0).setId(slideId);
+    }
+
+    @Test
+    public void test_getSlideSequence_slideExistsInSequence() {
+
+        // Positive scenario - Slide present in Sequence
+
+        sequenceObj.setId(sequenceId);
+        sequenceObj.setSlides(slidesList);
+        List<Sequence> sequencesList = new ArrayList<>();
+        sequencesList.add(sequenceObj);
+        Mockito.when(sequenceRepo.findSequencesForModule(moduleId)).thenReturn(sequencesList);
+        List<Sequence> slideSequencePresent = slideManagerToTest.getSlideSequences(slideId, moduleId);
+
+        String actualSlideIdSequence = slideSequencePresent.get(0).getSlides().get(0).getId();
+        String expectedSlideIdSequence = sequencesList.get(0).getSlides().get(0).getId();
+
+        Assert.assertEquals(actualSlideIdSequence, expectedSlideIdSequence);
+        Assert.assertEquals(slideSequencePresent.size(), sequencesList.size());
+
+    }
+
+    @Test
+    public void test_getSlideSequence_slideNotExistsInSequence() {
+
+        sequenceObj.setId(sequenceId);
+        sequenceObj.setSlides(slidesList);
+        List<Sequence> sequencesList = new ArrayList<>();
+        sequencesList.add(sequenceObj);
+        Mockito.when(sequenceRepo.findSequencesForModule(moduleId)).thenReturn(sequencesList);
+
+        List<Sequence> slideSequencePresent = slideManagerToTest.getSlideSequences(slideId, moduleId);
+        List<Sequence> actualSequenceSlideListNotPresent = slideManagerToTest.getSlideSequences(slideIdNotInSequence,
+                moduleId);
+        Assert.assertNotEquals(slideSequencePresent, actualSequenceSlideListNotPresent);

```

Comment:
I'm confused what are you comparing here? If the slide is not in any sequence, you expect the getSlideSequence to return an empty list, don't you? Why would you compare that to a different list? Espeically one you are also geetting from the method you want to test. If there's a bug, you can't trust any of these.